tests/test_black.py

```diff
@@ -209,8 +211,8 @@ def test_piping_diff(self) -> None:
             r"(STDIN|STDOUT)\t\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d\.\d\d\d\d\d\d"
             r"\+\d\d:\d\d"
         )
-        source, _ = read_data("simple_cases", "expression.py")
-        expected, _ = read_data("simple_cases", "expression.diff")
+        source, _ = read_data("cases", "expression.py")
```

Comment:
Possibly the test cases in this file that use `read_data()` should be cleaned up, especially those using the `cases` directory, but that can wait for another PR.